src/EnergyPlus/PlantLoopHeatPumpEIR.hh

```diff
@@ -197,6 +197,97 @@ namespace EIRPlantLoopHeatPumps {
         void oneTimeInit(EnergyPlusData &state) override;
     };
 
+    struct EIRFuelFiredHeatPump : public EIRPlantLoopHeatPump
+    {
+        // fixed configuration parameters
+        // reference data
+        // curve references
+        // flow rate terms
+        // simulation variables
+        // topology variables
+        // counters and indexes
+        // logic flags
+        // a couple worker functions to easily allow merging of cooling and heating operations
+
+        // enum definitions for Fuel Fired only
+        enum class OATempCurveVar
```

Comment:
Maybe a little clarification here: If the question is about if this should be moved out---the current treatment looks like more reasonable arrangement to put everything together; and can be tugged away without being lost somewhere (probably not a big problem with help of VS IDE); and accessing the definition just needed on more scope resolution. So it might be choice---unless we already have a convention about how to treat this scenario.

Or if the question is if these definition should be placed in the parent struct (EIRPlantLoopHeatPump)---then it should not, as they will only be used in the newly added FF Heat Pump and do not need to be inherited from the base class where they are not needed there.

Reply:
I don't know what the right guidance is for this. There is no problem with declaring a type inside a class scope, you just have to resolve the class name if you want to access the type outside of a class member function. We don't do this regularly which is why it looks weird, but it's generally a fine thing to do.

Reply:
Ok. This can stay as-is then.